public/math-service.js

```diff
@@ -2,11 +2,11 @@
 //   współrzędną X
 //   współrzędną Y
 const evaluateBoatDesireVector = (initialAngle) => {
-  const {boatSpeed} = getParams();
-  return ({
-    x: -boatSpeed * Math.cos(initialAngle),
-    y: boatSpeed * Math.sin(initialAngle),
-  });
+    const {boatSpeed} = getParams();
+    return ({
+        x: -boatSpeed * Math.cos(initialAngle),
+        y: boatSpeed * Math.sin(initialAngle),
+    });
```

Comment:
Zmień sobie proszę domyślne wcięcia, żeby tutaj nie zmieniać tego, co już istnieje

Reply:
Albo dobra, ja to cofnę